source/includes/write/bulk-write.scala

```diff
@@ -0,0 +1,91 @@
+import org.mongodb.scala._
+import org.mongodb.scala.bson.Document
+import org.mongodb.scala.model.Filters.equal
+import org.mongodb.scala.model.Updates.set
+import org.mongodb.scala.model._
+
+object BulkWrite {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+
+    // start-db-coll
+    val database: MongoDatabase = mongoClient.getDatabase("sample_restaurants")
+    val collection: MongoCollection[Document] = database.getCollection("restaurants")
+    // end-db-coll
+
+    {
+      // start-bulk-insert-one
+      val insertOneModel = InsertOneModel(
+          Document("name" -> "Blue Moon Grill",
+                   "borough" -> "Brooklyn",
+                   "cuisine" -> "American")
+      )
+      // end-bulk-insert-one
+
+      // start-bulk-update-one
+      val updateOneFilter = equal("name", "White Horse Tavern")
+      val updateOneDoc = set("borough", "Queens")
+      val updateOneModel = UpdateOneModel(updateOneFilter, updateOneDoc)
+      // end-bulk-update-one
+
+      // start-bulk-update-many
+      val updateManyFilter = equal("name", "Wendy's")
+      val updateManyDoc = set("cuisine", "Fast food")
+      val updateManyModel = UpdateOneModel(updateManyFilter, updateManyDoc)
+      // end-bulk-update-many
+
+      // start-bulk-replace-one
+      val replaceFilter = equal("name", "Cooper Town Diner")
+      val replaceDoc = Document("name" -> "Smith Town Diner",
+                                "borough" -> "Brooklyn",
+                                "cuisine" -> "American")
+      val replaceOneModel = ReplaceOneModel(replaceFilter, replaceDoc)
+      // end-bulk-replace-one
+
+      // start-bulk-delete-one
+      val deleteOneModel = DeleteOneModel(equal("name", "Morris Park Bake Shop"))
+      // end-bulk-delete-one
+
+      // start-bulk-delete-many
+      val deleteMany = DeleteManyModel(equal("cuisine", "Experimental"))
```

Comment:
S: [nit] use the same variable naming pattern as the other examples
```suggestion
      val deleteManyModel = DeleteManyModel(equal("cuisine", "Experimental"))
```